Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabFocusNotificationBus.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/EBus/EBus.h>
+
+namespace AzToolsFramework::Prefab
+{
+    class PrefabFocusNotifications : public AZ::EBusTraits
+    {
+    public:
+        virtual ~PrefabFocusNotifications() = default;
```

Comment:
update: Please make bus destructors protected and non-virtual - we're never going to delete an object through a bus interface so the destructor does not need to be virtual and making it protected prevents the unintentional use (AZ::EBusTraits actually does this too)

Reply:
Done!
Note that I based this on existing buses, so I'm not sure what the recommendations are. But I'll change these to protected.

Reply:
Awesome thanks! 🙂 (the other ones are technically wrong in my opinion, this should definitely be the preferred way)

Reply:
Good to know, will keep in mind :D